src/main/java/com/practise/tdd/PerfectNumberValidator.java

```diff
@@ -2,4 +2,8 @@
 
 public class PerfectNumberValidator {
 
+	public boolean isPerfectNumber() {
+		return false;
+	}
+
```

Comment:
Yes, that was a miss because I was constantly trying to remember to check in before making it green.